src/DataWrangling/JRA55/JRA55.jl

```diff
@@ -1,6 +1,6 @@
 module JRA55
 
-export JRA55FieldTimeSeries, JRA55PrescribedAtmosphere, JRA55RepeatYear
+export JRA55FieldTimeSeries, JRA55PrescribedAtmosphere, JRA55RepeatYear, JRA55MultipleYears
```

Comment:
Do we like this or

* RepeatYearJRA55
* MultiYearJRA55

the latter is how we say it out loud

Reply:
I'm not oppposted to "multi-year" rather than "multiple years" because "multi-year" is how we pronounce it